modules/cats-effect/src/main/scala/scalacache/CatsEffect.scala

```diff
@@ -16,6 +16,14 @@ object CatsEffect {
       val M: Async[IO] = asyncForCatsEffectAsync[IO]
```

Comment:
IMO the old `Mode[IO]` should be removed in favor of the Effectul type class.

But this would impact users that do not have the `import cats.effect.implicits._` in the scope.